harness/tests/integration_cases/test_raft.rs

```diff
@@ -5839,3 +5839,19 @@ fn test_fast_log_rejection() {
         assert_eq!(msgs[0].index, next_append_index, "#{}", i);
     }
 }
+
+#[test]
+fn test_turn_off_check_quorum() {
+    let l = default_logger();
+    let mut sm = new_test_raft(1, vec![1, 2, 3], 5, 1, new_storage(), &l);
+
+    sm.set_check_quorum(false);
```

Comment:
It's false by default. Perhaps check the differences between enabling and disabling `check_quorum`.

Reply:
Done